packages/bls12-381/bls12-381.6.0.1/opam

```diff
@@ -0,0 +1,46 @@
+opam-version: "2.0"
+synopsis: """\
+Implementation of BLS12-381 and some cryptographic primitives built
+on top of it"""
+maintainer: "Danny Willems <be.danny.willems@gmail.com>"
+authors: "Danny Willems <be.danny.willems@gmail.com>"
+license: "MIT"
+homepage:
+  "https://nomadic-labs.gitlab.io/cryptography/ocaml-bls12-381/bls12-381/"
+bug-reports:
+  "https://gitlab.com/nomadic-labs/cryptography/ocaml-bls12-381/issues"
```

Comment:
```suggestion
homepage:
  "https://nomadic-labs.gitlab.io/cryptography/ocaml-bls12-381/bls12-381/"
bug-reports:
  "https://gitlab.com/nomadic-labs/cryptography/ocaml-bls12-381/issues"
dev-repo:
  "git+https://gitlab.com/nomadic-labs/cryptography/ocaml-bls12-381.git"
```

Because the CI is complaining about
```
2022-12-20 18:02.31: Error in bls12-381.6.0.1:            warning 37: Missing field 'dev-repo'
```